app/workers/process_oci_image_worker.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+require 'minitar'
+
+require Rails.root / 'lib/oci_image_layout'
+
+class ProcessOciImageWorker < BaseWorker
+  MIN_TARBALL_SIZE  = 512.bytes     # to avoid processing empty or invalid tarballs
+  MAX_TARBALL_SIZE  = 512.megabytes # to avoid downloading excessive tarballs
```

Comment:
This is unlikely to be sufficient for some applications, but I feel like it's a good default. Maybe bumping it up to 1GB would be fine, since I've seen quite a few 800MB images. I think we should eventually add a max artifact size to accounts.

Reply:
E.g. Sentry is 250MB and PostHog is 1.5GB.

Reply:
Maybe add `max_upload` for individual uploads, and `max_ingress` and `max_egress` for monthly ingress/egress limits (which can be handled like `max_reqs`).